app/presenters/tree_node/orchestration_template.rb

```diff
@@ -1,9 +1,7 @@
 module TreeNode
   class OrchestrationTemplate < Node
-    set_attribute(:image) do
-      suffix = @object.class.name.underscore.split("_").last.downcase
-      suffix = 'vapp' if suffix == 'template'
-      "100/orchestration_template_#{suffix}.png"
+    set_attribute(:icon) do
+      "product product-template"
```

Comment:
wanted to say we're losing the type distinction here .. but looking at `100/orchestration_template_*.png` they all look the same, so not saying anything :)